### ISSUE TITLE
Homematic IP Devicename needs to be corrected

### ISSUE THREAD

**Issue by tkoroscha** [danielperna84/pyhomematic#110]
due to case sensitivity, devicetypes under pyhomatic/devicetypes/* should be corrected for Homematic-IP Types.

Wrong: HMIP-nnnn
correct: HmIP-nnnn

after that, events like Buttons, sabotage, etc. will be handeld correctly.

**Comment by danielperna84** [comment]
This is dependet of particular devices. Some are called `HMIP`, some `HmIP`. Sadly eq3 is being inconsitent with this. Which device are you referring to?

**Comment by tkoroscha** [comment]
a bit later, I recognized also, that inconsistency, cause I have HMIP-WRC2 and HmIP-WRC6, but the incosistency is not wanted and seems to be well known.
If you check devconfig, Devicetypes, you will get a list of all supported Types:

> HmIP-ASIR |   | ASIR
> HmIP-BBL |   | HmIP-BBL
> HmIP-BDT |   | BDT
> HmIP-BROLL |   | HmIP-BROLL
> HmIP-BSM |   | BSM
> HmIP-BWTH |   | WTH
> HmIP-BWTH24 |   | WTH
> HmIP-eTRV |   | TRV
> HMIP-eTRV |   | TRV
> HmIP-eTRV-2 |   | TRV
> HmIP-eTRV-2-UK |   | TRV-UK
> HmIP-FAL230-C10 |   | FAL230-C10
> HmIP-FAL230-C6 |   | FAL230-C6
> HmIP-FAL24-C10 |   | FAL24-C10
> HmIP-FAL24-C6 |   | FAL24-C6
> HmIP-FBL |   | HmIP-FBL
> HmIP-FDT |   | FDT
> HmIP-FROLL |   | HmIP-FROLL
> HmIP-FSM |   | FSM
> HmIP-FSM16 |   | FSM16
> HmIP-KRC4 |   | KRC4
> HmIP-KRCA |   | KRCA
> HmIP-MIOB |   | MIOB
> HmIP-MOD-OC8 |   | HmIP-MOD-OC8
> HmIP-PCBS |   | HmIP-PCBS
> HmIP-PCBS-BAT |   | HmIP-PCBS
> HmIP-PDT |   | PDT
> HmIP-PDT-UK |   | PDT
> HmIP-PS |   | PS
> HMIP-PS |   | PS
> HMIP-PSM |   | PSM
> HmIP-PSM |   | PSM
> HmIP-PSM-CH |   | PSM-CH
> HmIP-PSM-IT |   | PSM-IT
> HmIP-PSM-PE |   | PSM-PE
> HmIP-PSM-UK |   | PSM-UK
> HmIP-RC8 |   | RC8
> HmIP-SAM |   | HmIP-SAM
> HmIP-SMI |   | SMI
> HmIP-SMO |   | SMO
> HmIP-SMO-A |   | SMO
> HmIP-SPI |   | HmIP-SPI
> HmIP-SRH |   | SRH
> HmIP-STH |   | STH
> HmIP-STHD |   | STHD
> HmIP-STHO |   | HmIP-STHO
> HMIP-SWDO |   | SWD
> HmIP-SWDO |   | SWD
> HmIP-SWDO-I |   | SWD
> HmIP-SWSD |   | SWSD
> HMIP-WRC2 |   | WRC2
> HmIP-WRC2 |   | WRC2
> HmIP-WRC6 |   | WRC6
> HMIP-WTH |   | WTH
> HmIP-WTH |   | WTH
> HMIP-WTH-2 |   | WTH
> HmIP-WTH-2 |   | WTH
> 

This list figures out, that types in capital letters are the inconcistence naming. For example HMIP-eTRV and also HmIP-eTRV!
I would suggest using the same list, cause it may be possible, that eq3 changes names to consistence scheme, by having these devices in small letters and capital letters enables a dowward compatibility.

**Comment by danielperna84** [comment]
> If you check devconfig, Devicetypes, you will get a list of all supported Types:

What are you referring to? Is that some place at the CCU? I haven't found such a list yet.

**Comment by tkoroscha** [comment]
yes, a hidden tool, which s quite useful!

https://homematic.simdorn.net/verstecktes-config-tool/

**Comment by danielperna84** [comment]
Good to know. So essentially it would be best wo work through your list and add all known variants to the appropriate devicetype. The ones that are there already, are ones that are known to work. Adding the missing ones would then enable support for those where multiple variants are available.
Some time back we actually converted those tags to lower case, to not run into such issues. But that resulted in other problems, which is why we went with the case sensitive variant.
